src/router/index.js

```diff
@@ -25,9 +25,9 @@ const router = createRouter({
       component: Dashboards,
     },
     {
-      path: '/dashboards/dashboard',
-      name: 'dashboard',
+      path: '/dashboards/dashboard/:id',
```

Comment:
```suggestion
      path: '/dashboards/:id',
```
Following the same pattern in [Vue Router documentation](https://router.vuejs.org/guide/essentials/dynamic-matching.html#Dynamic-Route-Matching-with-Params), the inclusion of `/dashboard/` before the dynamic parameter `:id` to reference a specific object is unnecessary.

Reply:
Indeed it is unnecessary.